### PR TITLE
Data Extraction Maarten

### PR DESCRIPTION
**Pull request by mpsijm**
Tadaah, a bunch of raw data :smile: 

I also made two small other changes:
- Converted all tabs to spaces
- Renamed bibitem `@poo-caama2016a` to `@poo-caamano2016a` to be consistent with the author's name

### CODE REVIEW

**Comment by jessetilro** [comment]
Great work! Maybe you can update all mentions of "data synthesis" to "data extraction", because that is what we are currently doing.

**Comment by mpsijm** [comment]
Good point, I must have mixed up the two :slightly_smiling_face: Thanks! :smile:

File: 06-release-engineering-analytics.Rmd [hunk 1]
```diff
@@ -309,9 +315,396 @@ iteratively and planned for four weekly milestones.
 
 <!-- TODO -->
 
-## Appendix
+## Raw extracted data
```
Comment (mpsijm) [inline 06-release-engineering-analytics.Rmd:341]: See https://github.com/nsalminen/software-analytics-book/pull/8#discussion_r221455592

Change "Raw extracted data" to "Extracted Data"?

Comment (jessetilro) [reply 06-release-engineering-analytics.Rmd:341]: The raw part does not really matter, as long as it mentions "extracted" instead of "synthesized".

Comment (mpsijm) [reply 06-release-engineering-analytics.Rmd:341]: Okay, then I'll leave it like this :slightly_smiling_face: Thanks again for the feedback! :smile:

**Pull request merged**